CMakeLists.txt

```diff
@@ -390,82 +390,6 @@ if(gRPC_BUILD_GRPCPP_OTEL_PLUGIN)
   include(cmake/opentelemetry-cpp.cmake)
 endif()
 
-# Setup external proto library at third_party/envoy-api with 2 download URLs
```

Comment:
This section is not supposed to changes. (also it doesn't appear to have test coverage to catch this one) Would you take a look at why?

Reply:
This is the result of running the commands mentioned in https://github.com/grpc/grpc/pull/35272#issuecomment-1930284024.

Reply:
Yes but it shouldn't change so I guess this is a side effect of this PR. This code https://github.com/grpc/grpc/blob/b238652227dc39f8c81f6b9c72ef68809c8c9e25/tools/buildgen/extract_metadata_from_bazel_xml.py#L1054-L1059 seems relevant to this effect.

Reply:
Yeah, there is no `//external:` any more as this would be created by `bind()`.

Reply:
Oh I actually tried this in the PR but it doesn't seem to work... Let me try to debug the bazel query.

Reply:
The query to `//external:*` still works, so reverting the change in this file and then running the scripts also reverts the changes to this file.